packages/rancher-monitoring/prometheus-federator/generated-changes/patch/charts/helmProjectOperator/values.yaml.patch

```diff
@@ -1,17 +1,17 @@
 --- charts-original/charts/helmProjectOperator/values.yaml
 +++ charts/charts/helmProjectOperator/values.yaml
-@@ -129,8 +129,8 @@
- replicas: 1
+@@ -140,8 +140,8 @@
+   detectK3sRke2: false
  
  image:
--  repository: ghcr.io/rancher/helm-project-operator
--  tag: v0.3.0
-+  repository: rancher/prometheus-federator
-+  tag: ''
+-  registry: ghcr.io
+-  repository: rancher/prometheus-federator/helm-project-operator
++  registry: ''
++  repository: ''
```

Comment:
Could be looking at this wrong, but doesn't look right. We should be setting this (I think)

Reply:
I opted to set these as empty - as this completely will exclude this image from release artifacts (like `rancher-image-origins.txt`). Since we never want end-users touching this helps remove those remaining traces of them.

For end users, the values for this should only come from the main defaults in `prometheus-federator/values.yaml` or their specific values. Where as the origin `helm-project-operator` chart defaults to the `ghcr.io` container for HPO.

Reply:
Example: https://github.com/rancher/rancher/releases/tag/v2.10.1-alpha1
Grab `rancher-images-origins.txt` from the release and it will have the HPO image listed.

By removing these from values at `rancher/charts` repo it will prevent the HPO image from ending up in releases or security scans. However the PromFed images customers actually use will still be in both.